Malware Blog (#2919)
alcionai/corso: website/blog/2023-3-23-malware.md

---
slug: malware-detection
title: "Microsoft 365, Malware 👾, and your responsibilities"

> description: "By watching my

Reviewer: Is this the description we want here?

authors: nica
tags: [corso, microsoft 365, backups, security, malware]
date: 2023-3-23
image: ./images/invaders.png
---

![a clone of the game 'space invaders' Cover image By Lee Robinson - https://github.com/leerob/space-invaders, MIT, https://commons.wikimedia.org/w/index.php?curid=127314893](./images/invaders.png)

When we evaluate malware threats, we often think mainly of protecting our users.
The biggest concern is always going to be lost availability and leaked data if
malware affects our system. But like any threat with an infection model, part of
the story is about your responsibilities as an operations engineer to keep
others safe.
<!-- truncate -->

## Microsoft-hosted Malware

Research earlier in 2023 showed that Microsoft OneDrive was
[host to about 30% of all malware](https://www.cybertalk.org/2023/01/27/do-you-use-onedrive-or-google-drive-watch-out-for-this-malware/).
OneDrive is a popular platform for hosting malware because the malicious actor
can get a legitimate looking URL that will increase the chance of their payload
being downloaded or executed.

This malware hosting is usually done on accounts created by malicious actors,

> but it's even more effective if an compromised account within a legitimate

Reviewer: ```suggestion
but it's even more effective if a compromised account within a legitimate
```

organization can be used

The responsibility for addressing this issue lies more with administrators than

> with Microsoft. Data on OneDrive is after all customer data so at the end of the
> day MSFT it's not likely that MSFT will automatically start taking down files.  

Reviewer: ```suggestion
with Microsoft. Data on OneDrive is customer data ant it will be very intrusive and disruptive 
for Microsoft to automatically start taking down files.  
```

Anyone running OneDrive and SharePoint should take measures to detect and remove

> malware - to protect their own users and the broader community

Reviewer: ```suggestion
malware - to protect their own users and the broader community.
```

If you accept that using Microsoft OneDrive and 365 means that you should ‘be
part of the solution’ how can you take a stand against malware?

## Scan for Malware

Sophisticated malware is difficult to engineer. Threats like BazarLoader, which
use a Trojan horse to create an ISO which waits for the user to open an
innocent-looking ‘Documents’ folder, aren’t being developed from scratch every
day. Therefore, it’s possible to scan for malware and find most threats before
they affect large numbers of systems.

While there are a number of tools to scan backups, attachments, and other file
locations, I’m pleased to say that Corso has implemented
[malware scanning for your backups as of V0.5.0](https://github.com/alcionai/corso/releases/tag/v0.5.0).
Corso aims to prevent content already flagged as malware from making it in your
backups. Since Corso is free and open-source, admins can take advantage of this
and take action (for example delete, extract for forensic analysis) against files
flagged by Corso.

## See it in Action: Create a Malware-Free Backup with Corso

> We can hope that the first time you use a tool like Corso to scan your backups, you

Reviewer: ```suggestion
We hope that the first time you use a tool like Corso to scan your backups, you
```
